meta-lxatac-software/recipes-rust/bottom/bottom_0.6.8.bb

```diff
@@ -182,7 +182,170 @@ SRC_URI += " \
     crate://crates.io/winapi/0.3.9 \
 "
 
-
+SRC_URI[addr2line-0.16.0.sha256sum] = "3e61f2b7f93d2c7d2b08263acaa4a363b3e276806c68af6134c44f523bf1aacd"
```

Comment:
Could you add the command you used for the update to the commit message?

Reply:
Sure!
The `cargo.bbclass` makes the process quite straight forward. Just build the recipe without the hashes, then `cargo.bbclass` will make the build fail and print out a list of hashes to copy and paste.